earthpy/mask.py

```diff
@@ -161,10 +160,14 @@ def _apply_mask(arr, input_mask):
 
 
 def mask_pixels(arr, mask_arr, vals=None):
-    """Take an input array to be masked, single band mask layer such as a
-    pixel_qa layer for MODIS or Landsat, and apply a mask given a range of
-    values to mask. This function can also be passed a previously created
-    mask in place of a qa layer and a list of values.
+    """Apply a mask to an input array.
+
```

Comment:
How about:
`The mask_pixels function takes an input 1-dimensional masked array (mask_arr) and masks values in an n-dimensional input array. If mask_arr is provided in a boolean format, it is used as a mask. If mask_arr is provided as a non-boolean format and vals are provided, the function first masks values in the 1-dimensional mask_arr and then uses that mask_arr to mask the input arr. This function is useful when masking cloud and other unwanted pixels (e.g. pixel qa layers) found in landsat and other remote sensing data sets.   `

i dont love this text above btw but it's a start maybe?